apps/web/src/components/submission/components/Credential.tsx

```diff
@@ -221,6 +212,16 @@ const SpecialtySelector: React.FC<SpecialtySelectorProps> = ({
             />
           ))}
         </Select>
+        {enableDelete ? (
```

Comment:
if we want disallow deleting the last specialty, can we just check if index is 0?

Reply:
We want to make sure that you can delete any specialty selection, unless there is only one left.

Reply:
We want to make sure that you can delete any specialty selection, unless there is only one left.